server/form-pages/placement-application/request-a-placement/checkYourAnswers.ts

```diff
@@ -0,0 +1,36 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { Page } from '../../utils/decorators'
+
+import TasklistPage from '../../tasklistPage'
+import { PlacementApplication } from '../../../@types/shared'
+
+@Page({ name: 'check-your-answers', bodyProperties: ['reviewed'] })
+export default class Review implements TasklistPage {
+  name = 'check-your-answers'
+
+  title = 'Check your answers'
+
+  placementApplication: PlacementApplication
+
+  constructor(public body: { reviewed?: string }, placementApplication: PlacementApplication) {
+    this.placementApplication = placementApplication
+  }
+
+  previous() {
+    return 'updates-to-application'
+  }
+
+  next() {
+    return ''
+  }
+
+  response() {
+    return {}
+  }
+
+  errors() {
```

Comment:
Should we ensure they've checked the box here?

Reply:
We could add the validation but it wouldn't be handled here (agree this is confusing) - it's handled in the placementRequestController's submit method

Reply:
D'oh, course! 🤦